### PR TITLE
sandbox/apparmor: use kernel features with host apparmor_parser

### PR DESCRIPTION
**Pull request by zyga**

### CODE REVIEW

File: sandbox/apparmor/apparmor.go [hunk 1]
```diff
@@ -486,7 +486,7 @@ func AppArmorParser() (cmd *exec.Cmd, internal bool, err error) {
 	for _, dir := range filepath.SplitList(parserSearchPath) {
 		path := filepath.Join(dir, "apparmor_parser")
 		if _, err := os.Stat(path); err == nil {
-			return exec.Command(path), false, nil
+			return exec.Command(path, "--policy-features=<kernel>"), false, nil
```
Comment (bboozzoo) [inline sandbox/apparmor/apparmor.go:489]: this needs a long comment explaining `<..>` meaning.

Comment (zyga) [reply sandbox/apparmor/apparmor.go:489]: https://gitlab.com/apparmor/apparmor/-/blob/master/parser/parser_main.c#L638

Comment (zyga) [reply sandbox/apparmor/apparmor.go:489]: I agree that we need a better grasp of what the options mean. There are more things we could pass and I'm not sure which ones we really should and should not touch.

Comment (zyga) [reply sandbox/apparmor/apparmor.go:489]: https://gitlab.com/apparmor/apparmor/-/commit/162da1ba48635a3c0be8390bd405105d700c715f gives some insight

**Comment by zyga** [comment]
I've rebased this on up-to-date master to see what kind of failures remain with all the other fixes that got merged.

**Comment by zyga** [comment]
There's an alternative approach in https://github.com/snapcore/snapd/pull/13707

**Comment by zyga** [comment]
Closed in favor of https://github.com/snapcore/snapd/pull/13711

**Pull request closed without merging**